configs/config.go

```diff
@@ -16,14 +18,41 @@ type API struct {
 }
 
 type Terraform struct {
-	Bin    *string
-	Source *string
+	Bin            *string
+	Source         *string
+	TFInitOptions  *string `yaml:"tfInitOptions,omitempty"`
```

Comment:
`--tf-init-options` makes sense as a name for a CLI flag, but I wonder if it's a little redundant for the config file, just in that it's already namespaced under `terraform`.

What you do think about just `initOptions` or even just `init`?

Reply:
Totally cool with `initOptions`